pkg/devfile/parser/data/2.0.0/components_test.go

```diff
@@ -0,0 +1,54 @@
+package version200
+
+import (
+	"testing"
+
+	common "github.com/openshift/odo/pkg/devfile/parser/data/common"
+)
+
+func TestGetCommands(t *testing.T) {
```

Comment:
Can you please add some more comment for this so that it will be better to understand.